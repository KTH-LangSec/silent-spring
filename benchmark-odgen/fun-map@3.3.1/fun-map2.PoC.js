const { assocInM } = require('./fun-map@3.3.1')
assocInM({}, ['__proto__', 'polluted', 'prop'], 'yes')