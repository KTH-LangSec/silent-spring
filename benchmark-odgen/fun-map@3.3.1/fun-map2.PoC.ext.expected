fun-map@3.3.1/index.js:143:24